package.json

```diff
@@ -20,6 +20,10 @@
     "not ie < 11",
     "not IE_Mob 11"
   ],
+  "engines": {
```

Comment:
do we need this to overcome some CI issues? otherwise, maybe minor/major version is a better to to add this kind of package field

Reply:
This is formallly documenting/enforcing our CURRENT minimum node version. We plan to up the minimum version in the next major release